### DISCUSSION TITLE
AAWireless TWO

### DISCUSSION THREAD

**Discussion by JVKeller** [AndreyPavlenko/Fermata#552] [General]
Has anyone been able to get one of the new AAWireless TWO dongles and test it out?

I almost ordered one this week, but figured I better pay my electric bill instead 🙄😒

**Comment by djak250** [comment]
It works. I just set one up.

**Comment by JVKeller** [comment]
Yep. Finally got my hands on one. Works great! 

Gotta figure out a way to get the regular fermata app to use the full screen. If I use screen mirroring, I loose D-pad control.

**Comment by JVKeller** [comment]
I'll also add, the lag on videos is much better! It's not in perfect sync, but it's way closer than it was.

**Reply by drrules** [reply]
AAWireless Two with A14 or A15?
